lib/reek/smells/smell_repository.rb

```diff
@@ -25,6 +25,7 @@ def self.smell_types
       # @return [Array<Reek::Smells::SmellDetector>] All SmellDetectors that we want to filter for
       #         e.g. [Reek::Smells::Attribute].
       def self.eligible_smell_types(filter_by_smells = [])
+        filter_by_smells = Array(filter_by_smells)
```

Comment:
>> I do this to ensure we have an array, so the #include? later on uses the Array version, rather than the String version. 

Ok, maybe I should phrased that better :)
My question was of provocative nature. I understand why you're doing this but it feels erratic. Here we would do this but there are another 100 places in our code where we're not doing this. 
I'd prefer to remove it.